sklearn/model_selection/tests/test_split.py

```diff
@@ -1771,16 +1771,15 @@ def test_nested_cv():
 
     cvs = [
         LeaveOneGroupOut(),
-        LeaveOneOut(),
-        GroupKFold(n_splits=3),
-        StratifiedKFold(),
-        StratifiedGroupKFold(n_splits=3),
-        StratifiedShuffleSplit(n_splits=3, random_state=0),
+        GroupKFold(n_splits=2),
+        StratifiedKFold(n_splits=2),
+        StratifiedGroupKFold(n_splits=2),
+        StratifiedShuffleSplit(n_splits=2, random_state=0),
     ]
 
     for inner_cv, outer_cv in combinations_with_replacement(cvs, 2):
         gs = GridSearchCV(
-            Ridge(), param_grid={"alpha": [1, 0.1]}, cv=inner_cv, error_score="raise"
+            Ridge(max_iter=1), param_grid={"alpha": [1, 0.1]}, cv=inner_cv, error_score="raise"
```

Comment:
I would not change the number of iteration here. It will generate some new `ConvergenceWarning`. Reducing the number of splits seem the best.

Reply:
Another possibility would be to use a `DummyClassifier` and tune two simple strategies. It will be extremely fast and will be enough for this smoke test.

Reply:
Hi @glemaitre I see there that I had called the Ridge() twice and I realized that this will definitely fail. I did this late last night therefore I seem to missed that. Apologies editing the file now.